interface-elements-for-web/articles/grid/customization-dialog/grouping-page.md

```diff
@@ -4,10 +4,12 @@ author: Anna Kondratova
 legacyId: 118821
 ---
 # Grouping Page
-Select a column in the "Columns to Group" list to group grid data.  The grid data can be grouped by more than one column, and a list of the grouped columns is displayed at the top of **Grouping** page.
+The **Grouping** page enables you to group grid data and change the order of grouping columns within the grid.
```

Comment:
grouping columns?

Reply:
Grouping columns are columns that are used to classify data into categories of information. Data rows are grouped by grouping columns. We use this term in technical documentation. Perhaps:

"You can use the Grouping page to group grid data and change grouping priority order within the grid."